### DISCUSSION TITLE
[Projects Beta] Field level permissions

### DISCUSSION THREAD

**Discussion by jwindmuller** [community/community#12220] [Projects and Issues]
Allowing more granular access control over fields on the new projects would give developers freedom to work on the boards and issues without danger of them changing properties they shouldn't. Some examples:

1. I want to be able to let "Read" users at the project level to still be able to move items on the board as they make progress on them. 
2. But I don't want to allow developers to add or remove issues from a sprint
3. Similarly, I don't want to let them change the issue story points

**Comment by brettwillis** [comment]
Similar to #7478
